rtc_data_service/src/attestation/service.rs

```diff
@@ -0,0 +1,89 @@
+use std::convert::TryInto;
+
+use actix::{Addr, MailboxError};
+use actix_web::{error::ErrorInternalServerError, post, web, HttpRequest};
+use models::*;
+use rtc_types::{AttestError, AttestationResponse};
+use rtc_uenclave::AttestationError;
+
+use crate::data_enclave_actor::DataEnclaveActor;
+use crate::merge_error::*;
+
+use super::AttestationMessage;
+
+#[post("auth/attest")]
+pub async fn req_attestation_jwt(
+    _req: HttpRequest,
+    req_body: web::Json<RequestBody>,
```

Comment:
This call does not need a request body, and we should make this call a `get` call instead